app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -11,45 +11,15 @@ class GetUspsProofingResultsJob < ApplicationJob
 
   queue_as :long_running
 
-  def email_analytics_attributes(enrollment)
-    {
-      timestamp: Time.zone.now,
-      user_id: enrollment.user_id,
-      service_provider: enrollment.issuer,
-      delay_time_seconds: mail_delivery_params[:wait],
-    }
-  end
+  rescue_from GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError do |exception|
+    analytics.idv_in_person_usps_proofing_results_job_terminated(
+      **enrollment_outcomes,
+      duration_seconds: (Time.zone.now - started_at).seconds.round(2),
+      error_class: exception.class.name,
+      error_message: exception.message,
+    )
 
-  def enrollment_analytics_attributes(enrollment, complete:)
-    {
-      enrollment_code: enrollment.enrollment_code,
-      enrollment_id: enrollment.id,
-      minutes_since_last_status_check: enrollment.minutes_since_last_status_check,
-      minutes_since_last_status_update: enrollment.minutes_since_last_status_update,
-      minutes_to_completion: complete ? enrollment.minutes_since_established : nil,
-      issuer: enrollment.issuer,
-    }
-  end
-
-  def response_analytics_attributes(response)
-    return { response_present: false } unless response.present?
-
-    {
-      fraud_suspected: response['fraudSuspected'],
-      primary_id_type: response['primaryIdType'],
-      secondary_id_type: response['secondaryIdType'],
-      failure_reason: response['failureReason'],
-      transaction_end_date_time: response['transactionEndDateTime'],
-      transaction_start_date_time: response['transactionStartDateTime'],
-      status: response['status'],
-      assurance_level: response['assuranceLevel'],
-      proofing_post_office: response['proofingPostOffice'],
-      proofing_city: response['proofingCity'],
-      proofing_state: response['proofingState'],
-      scan_count: response['scanCount'],
-      response_message: response['responseMessage'],
-      response_present: true,
-    }
+    raise exception
```

Comment:
Very minor nit but this seems more ruby-esque

```suggestion
    raise
```